controllers/discordactions.js

```diff
@@ -3,7 +3,6 @@ const admin = require("firebase-admin");
 const config = require("config");
 const jwt = require("jsonwebtoken");
 const discordRolesModel = require("../models/discordactions");
-const { retrieveUsers } = require("../services/dataAccessLayer");
```

Comment:
why exactly we are removing the data access layer ?

Reply:
This is not being used anymore.